api/TimeAddressableMediaStore.yaml

```
@@ -762,7 +762,7 @@ paths:
     delete:
       description: |
         Deletes the flow and associated segments. If flow segment deletion
-        takes too long then this request will redirect to a Flow Deletion Request
+        takes too long then this request will redirect to a Flow Delete Request via a "Location" header
```

The spec has this as returning a `202` with a `Location` header, but the implementation in Squirrel returns a 201 - I'm leaning towards Squirrel being incorrect here, and I've added a note to https://www.pivotaltracker.com/story/show/187691389 to fix that.

Nit: Not sure I like calling this a "redirect" since that implies a 3xx code. How's this?
```suggestion
        takes too long then this request will return 202 Accepted and the `Location` header will point to a
        Flow Delete Request to monitor deletion progress
```